middleware/validations/validateAll.js

```diff
@@ -7,6 +7,9 @@ const validateUser = [
   check('lastName', 'lastName is required')
     .not()
     .isEmpty(),
+  check('location', 'location is required')
```

Comment:
Hey @Kasulejoseph , could we not make it required, a user may decide to skip the step.